### ISSUE TITLE
Live load the changes to UFO

### ISSUE THREAD

**Issue by pathumego** [trufont/trufont#208]
When using in a workflow it is convieneient if changes to UFO by external programme can be detected and loaded. Prepolator dose this nicely.
<img width="439" alt="screen shot 2016-01-23 at 10 31 33 pm" src="https://cloud.githubusercontent.com/assets/6846609/12533229/242639ca-c221-11e5-9dfc-422fd0809665.png">

**Comment by adrientetar** [comment]
Right. Does it work on other parts of the font like font info etc.?

**Comment by pathumego** [comment]
Prepolator is only about fixing the master compatibility issues. There is no to inspect font info. But when I change fontinfo form FF it prompted some changes to glyphs. Probably caused by rewriting of UFO. I think defferenciating FontInfo and actual outline data would be better. I actually dont care much for FontInfo in reagrd to this feature.

**Comment by adrientetar** [comment]
I have something locally that paves the way for this already.

**Comment by pathumego** [comment]
Nice! Thanks a lot for time and effort you are putting in!  Once you have the extension spec ready we will have more actual code contributions coming in. Hope we could meet at LGM :)

**Comment by adrientetar** [comment]
Thanks. I don't know for certain whether I'll be there just yet but, sure.

**Comment by pathumego** [comment]
@adrientetar dose `File>Reload From Disk` supposed to reload the file? If so, its not working on Mac.

**Comment by adrientetar** [comment]
It works for me. Font has to have been saved as a UFO at least once.

**Comment by davelab6** [comment]
On 27 July 2016 at 10:05, Adrien Tétar <notifications@github.com> wrote:

> Font has to have been saved as a UFO at least once.


So if I open a file, make some edits, and want to reset it, it won't work?

**Comment by pathumego** [comment]
@adrientetar saving the file worked. Following up on a previous comment; Robofont dose this nicely, and it updates Font Info and other data too. And It gives a nice UI to review and reject or accept only selected changes. 

It would be nice to have a option in file menu to make TF update the font automatically, without needing the review. (turned off by default)

**Comment by pathumego** [comment]
@davelab6 reset works after first save.

**Comment by davelab6** [comment]
> reset works after first save.

This seems broken to me.

**Comment by davelab6** [comment]
(This is similar to https://github.com/fontforge/fontforge/issues/637)
